### ISSUE TITLE
[QUESTION] LICENSE

### ISSUE THREAD

**Issue by nicotravassos** [adrian-web/laracasts-forum#2]
Hi @adrian-web ,

I would like to use this repository within my project template I am building but unsure of licensing as my project is private. Is the licensing required to be displayed on my site under licensing, credits or is this not required and script is free to do whatever?

Please keep in mind that I will not be cloning this project and not including the commits - I will be copying most of it as I do not require some parts.

**Comment by adrian-web** [comment]
Hi @nicotravassos ,
I added MIT license to the project. Hope this solves your issue.

**Comment by nicotravassos** [comment]
Thanks @adrian-web judging by the license type using pieces of you code requires me to include you license with my code would you require me to have the licensing displayed within the frontend of my code under a credits section or is that not necessary?

**Comment by adrian-web** [comment]
No, it's not necessary to include anything about me or used license.
